usecases/objects/add.go

```diff
@@ -29,7 +29,7 @@ type schemaManager interface {
 	AddTenants(ctx context.Context, principal *models.Principal,
 		class string, tenants []*models.Tenant) (err error)
 	GetClass(ctx context.Context, principal *models.Principal,
-		name string,
+		name string, consistency bool,
```

Comment:
I won't expose the consistency flag to internal code because it's always false, which an be confusing and error prone. Instead we can create a new struct that exposes the exact same old interface (without the flag) for internal use (i.e. usecase/object/). This could simply be wrapper around the actual handler struct. This way, we ensure that internal code always uses the correct interface

Reply:
I agree with you, I'll change that and add a struct wraper around it. We can change it if/when we decided to add this consistency flag to other endpoints.